CHANGELOG.md

```diff
@@ -19,9 +19,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 <!-- ## [x.y.z] - YYYY-MM-DD -->
 
-## Unreleased
-
-<!-- ### Changed -->
+## [1.0.1] - 2022-09-12
```

Comment:
So, I think technically, while almost everything in this release was additive optional properties, the fix to add the `CustomElement` interface to `CustomElementDeclaration` added a required `isCustomElement` field to `CustomElementDeclaration` that didn't exist before in the schema.

If you search https://unpkg.com/custom-elements-manifest@1.0.0/schema.json for `"customElement"`,  you'll only find it under `MixinDeclaration`.

So _technically_ this could break someone who was making `CustomElementDeclarations` without that flag set, and maybe should be a breaking change?

Reply:
I think that from the JSON-schema side, this isn't a breaking change because you don't represent the type of object CustomElementDeclaration nominally, but they're validated structurally. In this case `declarations` has an anyof that includes CustomElementDeclaration. But CustomElementDeclaration shares a 'kind' with ClassDeclaration, and without `customElement: true`, there wasn't a good way to discriminate CustomElementDeclaration from ClassDeclaration.

So the old schema should validate (and the simple one in examples/ still does), but the class declarations in there would be interpreted as classes, not elements.

This was just a big bug in how the JSON schema was generated from the .d.ts file, and why I checked it in so we could review it.

Now on the TypeScript side, this is a breaking change as a small addition to the CustomElementDeclaration interface. You could declare a value as a CustomElementDeclaration and would get a new error. I don't know who's relying on the TypeScript side right now...

So I don't know. It wasn't really possible to generate a proper schema before.